src/elements/Shield/Shield.styl

```diff
@@ -66,11 +66,12 @@
 
   sizes('font-size')
 
-  height 2.25em
-
-  box-sizing border-box
 
+  height $height-control
+  width $width-control
+  box-sizing $box-sizing-control
   border var(--shield-border-width) solid var(--shield-primary)
```

Comment:
Since border is a different property from box dimensions, we must leave a blank line before it.